dttools/test/TR_mq_poll.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/sh
+
+. ../../dttools/test/test_runner_common.sh
+
+exe="mq_poll.test"
+
+prepare()
+{
+	${CC} -I../src/ -g $CCTOOLS_TEST_CCFLAGS -o "$exe" -x c - -x none ../src/libdttools.a -lm <<EOF
+#include <assert.h>
```

Comment:
I suggest making this example and the one below proper .c files. In that way it will be compiled with the flags as any other file. Otherwise, if we update ./configure, then we may also need to update this TR*.sh file.